### ISSUE TITLE
Main context menu need update

### ISSUE THREAD

**Issue by onemen** [onemen/TabMixPlus#46]
Error in the console from Firefox 91 - nightly
```
08:50:51.242 Tabmix:
TMP_updateMainContextMenu was unable to change gContextMenu.openLinkInTab.
can't find string
    /allowMixedContent:|charset:/

Try Tabmix latest development version from tabmixplus.org/tab_mix_plus-dev-build.xpi,
Report about this to Tabmix developer at http://tabmixplus.org/forum/
3 click.js:698:7
```
Screenshot Firefox 91 - nightly
![menu](https://user-images.githubusercontent.com/3650909/124374709-509bc600-dca6-11eb-92e5-35d4226afa62.png)

**Comment by 117649** [comment]
![图片](https://user-images.githubusercontent.com/8281345/124376527-f583c680-dcd9-11eb-918b-079879352627.png)
me on 90.0b12

**Comment by onemen** [comment]
Did you send it?
I'v got it only on my mail and not here ???

> ![124376527-f583c680-dcd9-11eb-918b-079879352627](https://user-images.githubusercontent.com/3650909/124423061-1b0de000-dd6d-11eb-92b1-03d9a7d64060.png)
> me on 90.0b12

**Comment by 117649** [comment]
> @117649 Did you send it?
> I'v got it only on my mail and not here ???
> 
> > ![124376527-f583c680-dcd9-11eb-918b-079879352627](https://user-images.githubusercontent.com/3650909/124423061-1b0de000-dd6d-11eb-92b1-03d9a7d64060.png)
> > me on 90.0b12
> 

Yes, but it was a mistake the error is from disable and enable TMP without restart. Just ignore it.

**Comment by onemen** [comment]
fixed by 4c6f156fd8e12471ce388d201ca22566df0e1623
